iridium-cli/src/assembly/bin.xml

```diff
@@ -46,6 +46,16 @@
             </includes>
         </fileSet>
 
+        <fileSet>
+            <directory>../conf</directory>
+            <outputDirectory>conf</outputDirectory>
+            <includes>
+                <include>*</include>
```

Comment:
Do you want everything or only those packages that are specified here?

Reply:
So far, yes.